cotainr/tests/container/patches.py

```diff
@@ -56,6 +56,17 @@ def mock_enter(self):
 
         return ret_val
 
+    def outside_container_create(self, env_file):
```

Comment:
This does not seem to be used anywhere. Am I missing something? And why is `patch_fake_singularity_sandbox_env_folder` still relevant?

Reply:
My bad. I just got confused by the way GitHub only shows some of the context around the change.